### DISCUSSION TITLE
Duda proyecto final

### DISCUSSION THREAD

**Discussion by HM9N** [rramosp/ai4eng.v1#261] [UDEA Students]
Buenos días

Me podrían explicar por favor a qué se refiere este enunciado, no me queda claro aún:

"Encontrar los mejores hiperparámetros para DOS combinaciones de algoritmo no supervisado + algoritmo predictivo"

Si pueden darme un ejemplo sería mejor.

Muchas gracias!

**Comment by imEag** [comment]
Hola! debes ensayar diferentes algoritmos y mirar cual combinación funciona mejor para tu proyecto. Por ejemplo, PCA con el n componets que mejor resultado + random forest con max depth con mejor resultado y luego ensayar pca + svm u otros.
